restapi/stories.go

```diff
@@ -81,7 +81,7 @@ func (api api) ReturnAllStories(w http.ResponseWriter, r *http.Request) render.R
 	var storiesByTags []models.Tag
 
 	getVisibility := r.URL.Query()["visibility"]
-	visibility := true
+	visibility := false
```

Comment:
```suggestion
	visibility := true
```
Since `stories` should return ALL stories now regardless of visibility

Reply:
it would not matter, but i changed it to true